core/jazz_assets/components/crud/getList.js

```diff
@@ -0,0 +1,94 @@
+ /**
+      Search a Asset-Catalog entry in dynamodb table
+      @module: getList.js
+      @description: CRUD functions for asset catalog
+      @author:
+      @version: 1.0
+  **/
+
+ const utils = require("../utils.js"); //Import the utils module.
+ const logger = require("../logger.js")(); //Import the logging module.
+
+ module.exports = (query, asset_table, onComplete) => {
+     // initialize dynamodb
+     var docClient = utils.initDocClient();
+
+     var filter = "";
+     var insertAndString = " AND ";
+     var attributeValues = {};
+     var attributeNames = {};
+
+     var params = {
```

Comment:
try using const/lets. ES6 stuff. Please update in other places as well

Reply:
updated